src/main/java/duke/Task.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+/** Task is an object that contains all information about a task
+ * @author CShuwen
+ * @version 1.0
+ * @since 0.0
+ */
+public class Task {
+    protected String description;
+    private int isDone;
```

Comment:
I suggest using `boolean` for isDone. The variable name may confuse other people thinking that it is a `boolean` based on the naming conventions from the coding standard.